controllers/courses.js

```diff
@@ -12,6 +12,8 @@ const logger = require('../helpers/logger');
 
 const router = express.Router();
 
+const { CALENDAR_SERVICE_ENABLED, HOST } = process.env;
```

Comment:
This is a feature, we use `FEATURE_...` named ENV's in other situations, why not here?

Reply:
I do not add this envs, only remove it from request callback into global scope.